charts/influxdb/README.md

```diff
@@ -228,8 +244,8 @@ spec:
         - name: influxdb-restore
           image: influxdb:1.7-alpine
           volumeMounts:
-            - name: backup
-              mountPath: /backup
+            - name: {{ include "influxdb.fullname" . }}-backups
+              mountPath: {{ .Values.backup.directory | quote }}
```

Comment:
this is not a helm template, it is an example raw yaml of a restore job, please revert